client/landing/gutenboarding/components/domain-picker/suggestion-item-placeholder.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@wordpress/components';
+
+const DomainPickerSuggestionItemPlaceholder: FunctionComponent< Button.AnchorProps > = props => {
+	return (
+		<Button className="domain-picker__suggestion-item" isTertiary { ...props }>
```

Comment:
This component doesn't receive any props, does it?

```suggestion
const DomainPickerSuggestionItemPlaceholder: FunctionComponent = () => {
	return (
		<Button className="domain-picker__suggestion-item" isTertiary>
```